community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
```

Comment:
> For example, the diversity of participants and viewpoints at a European event could be improved by funding the travel of someone from Canada. Is that potentially covered from this allocation?

Agreed that there is some ambiguity here that might make it difficult for the council to review applications. Is the diversity / underrepresentation applied to the country of destination of the event? Or is it applied from a global tech scene point of view?

Another example: A collaborator from Colombia may fit well the diversity definition from a conference organized in the global north but will every Colombian collaborator be eligible in case the conference is held in Colombia?

Reply:
25% is an allocation and we would not be limited to that amount, going over to support diverse candidates would be fantastic.

We were given specific legal guidance on what information we can collect regarding diversity. Additionally we were given guidance on what specific language to use. 

In this context diversity means identifying from an underrepresented group and this is open to interpretation by the submitter. Geographic location of the person is collected as part of the form and the CPC can and should factor this into their decisions.

Reply:
Is any guidance provided elsewhere to the applicants or the CPC about how eligibility for being diverse is determined? In particular, the questions @ruyadorno raises above about what the point of view is for determining diversity and underrepresentation don't seem addressed at all.

Reply:
This is my understanding, so I would love others @joesepi & @PaulaPaul to jump in if I state anything incorrectly.

We've tried to implement this in a way that the responsibility of deciding who is from an underrepresented group  does not fall to the CPC.  The form asks whether the applicant identifies as part of such a group. If they affirm, we accept their self-identification.

Reply:
As far as I recall, we decided not to be explicit on what being a diverse individual entitles and let people decide if they believe they belong to underrepresented groups. It is a vote of fairness and trust between the CPC and the applicant. Of course, if the system is abused (or we have reasons to believe a candidate is abusing the system), their travel fund request will (very possibly) be rejected.

Reply:
I get the idea of relying on self-identification for underrepresentation. But I'm still missing some clarity on whether the set of "diverse participants" is exactly equal to the set of people who "identify with an underrepresented group", or does it potentially include some other applicants as well?

If those two sets are the same, why are we using completely different words here and in the request form?

Reply:
These sets should be the same. We should apply the sMe wording all places to reduce confusion then

Reply:
Yep this is a really good catch. I'll fix the wording and then suggest we merge.

Additional comments/discussion we can spin out into individual issues.

Thanks for the help on this one!